SonarQube.Client.Tests/SonarQubeService_GetNotificationEventsAsync.cs

```diff
@@ -39,7 +39,7 @@ public async Task GetNotifications_ExampleFromSonarQube()
             var now = new DateTimeOffset(2017, 10, 19, 13, 0, 0, TimeSpan.FromHours(2));
 
             await ConnectToSonarQube("6.6.0.0");
-            SetupRequest("api/developers/search_events?projects=my_project&from=2017-10-19T13:00:00%2B0200",
+            SetupRequest("api/developers/search_events?projects=my_project&from=2017-10-19T13%3A00%3A00%2B0200",
```

Comment:
`:` was replaced with `%3A` after upgrading the package 🤷‍♀️

Reply:
Revert? I don't think we needed to change it

Reply:
We do, it fails otherwise